.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 .netrc
-moduleIndex.json
\ No newline at end of file
+moduleIndex.json
```

Comment:
Beware, still shows a change (I guess due to the line ending / newline)

Reply:
mmh can't see it in the files in this PR, can you verify?

Reply:
Probably because you have whitespaces hidden ;)

Reply:
![image](https://github.com/Azure/bicep-registry-modules/assets/5365358/66402185-7f2e-402e-8bbf-bad4627739ae)

Reply:
done